Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:3.1.2
+
+# System prerequisites
+RUN apt-get update \
+ && apt-get -y install build-essential libpq-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+# If you require additional OS dependencies, install them here:
+# RUN apt-get update \
+#  && apt-get -y install imagemagick nodejs \
+#  && rm -rf /var/lib/apt/lists/*
+
+ADD Gemfile /app/
+ADD Gemfile.lock /app/
+WORKDIR /app
+RUN bundle install
+
+ADD . /app
+
+# Collect assets. This approach is not fully production-ready, but
+# will help you experiment with Aptible before bothering with assets.
+# Review http://go.aptible.com/assets for production-ready advice.
+RUN set -a \
+ && . ./.aptible.env \
```

Comment:
The fact that this always includes `.aptible.env` means it's very hard to `docker build` on your own workstation. I find this to be the biggest hassle with `vita-min`'s Dockerfile.

https://deploy-docs.aptible.com/docs/docker-build-configuration

If the only real environment variable that matters is `RAILS_ENV=production` then I encourage you to use that, rather than include the `.aptible.env` goop. Example:

```RUN RAILS_ENV=production bundle exec rake assets:precompile```